soco/core.py

```diff
@@ -1527,9 +1484,9 @@ def get_favorite_radio_shows(self, start=0, max_items=100):
         """Get favorite radio shows from Sonos' Radio app.
 
         Returns:
-        A list containing the total number of favorites, the number of
-        favorites returned, and the actual list of favorite radio shows,
-        represented as a dictionary with `title` and `uri` keys.
+            dict: A dictionary containing the total number of favorites, the
+            number of favorites returned, and the actual list of favorite radio
```

Comment:
Same here. But I might be mistaken. If you have the compiled docs from this branch at hand, maybe you can just check if I'm correct